integration

```diff
@@ -531,6 +531,7 @@
   "msekoranja/emsc-hacs-repository",
   "msp1974/homeassistant-jlrincontrol",
   "mtarjoianu/ha_lektrico",
+  "mtrab/clever",
```

Comment:
<https://github.com/MTrab/clever/blob/master/hacs.json> Needs a country, this is for Denmark only.

Reply:
Added